_drafts/technical-workflow.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "The technical workflow of Etsy France"
+date:   2015-02-23 13:55:00
+categories: workflow
+tags: [workflow, jira, github, pr, review]
+author: antoine
+---
+
+Here at Etsy France we tried several organisations.  
+For some years now we fixed our organisation in scrum. To manage our workflow we use some useful tools that are interconnected: [Jira][jira], [Github][github], [HipChat][hipchat] and [Travis CI][travis].  
+This post will explain to you what is our daily workflow from the backlog to the deployment.
+
+## Our organisation
+
+In fact we do not have only one backlog, we have several backlogs because we do not have only one technical team but many.  
```

Comment:
In fact, we have more than just one backlog because we're split into several distinct technical teams.